profiles/common/modules/features/nexteuropa_trackedchanges/NETCInfo.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file
+ * Contains \NextEuropaTrackedChangesInfo.
+ */
```

Comment:
I agree but so far, it is blocked by the current coder rules. So, until we integrate the new coding standard rules, I fear we have no choice but keeping them.